### PR TITLE
Update README for upcoming v2.3.0 release

### PR DESCRIPTION
**Pull request by wiz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,18 +96,18 @@ Edit `mempool-config.json` to add your Bitcoin Core node RPC credentials:
   "MEMPOOL": {
     "NETWORK": "mainnet",
     "BACKEND": "electrum",
-    "HTTP_PORT": 8999,
-    "API_URL_PREFIX": "/api/v1/",
```
Comment (knorrium) [inline README.md:100]: this is still needed

File: README.md [hunk 2]
```diff
@@ -116,10 +116,6 @@ Edit `mempool-config.json` to add your Bitcoin Core node RPC credentials:
     "USERNAME": "mempool",
     "PASSWORD": "mempool",
     "DATABASE": "mempool"
-  },
-  "STATISTICS": {
```
Comment (knorrium) [inline README.md:120]: @wiz why are you removing this?

File: README.md (hunk 1 above)
Comment (wiz) [reply README.md:100]: this is not a full configuration file, it's just suggesting what to change from copying the full one

File: README.md (hunk 2 above)
Comment (wiz) [reply README.md:120]: most users won't disable it

**Review by softsimon — APPROVED** [review]
utACK

**Pull request merged**